net-dialup/freeradius/freeradius-3.0.17.ebuild

```diff
@@ -214,7 +219,14 @@ src_install() {
 	systemd_newtmpfilesd "${FILESDIR}"/freeradius.tmpfiles freeradius.conf
 	systemd_dounit "${FILESDIR}"/freeradius.service
 
-	find "${ED}" \( -name "*.a" -o -name "*.la" \) -delete || die
```

Comment:
We prefer this instead of prune_libtool_files

Reply:
Ok, i revert it but personally i prefer use of a specific function from an eclass, more easy to maintains, imho.